index.html

```diff
@@ -498,8 +507,20 @@ <h3>The <dfn>observe()</dfn> method</h3>
             </li>
             <li>
               [=Queue a global task=] on the [=PressureObserver task source=] given |document|'s
-              [=relevant global object=] |relevantGlobal| to run these steps:
+              [=relevant global object=] |relevantGlobal| to run these <dfn>finalize observe steps</dfn>:
               <ol>
+                <li>
+                  If |promise| was rejected, run the following substeps:
+                  <ol>
+                    <li>
+                      If |relevantGlobal|'s [=registered observer list=] for |source| is [=list/empty=],
+                      deactivate [=data delivery=] of |source| data to |relevantGlobal|.
+                    </li>
+                    <li>
+                      Abort the [=finalize observe steps=].
```

Comment:
Can't you just say "Return" without having to define "finalize observe steps"? Per https://infra.spec.whatwg.org/#algorithm-control-flow "Return" should _"terminate the algorithm the statement was in"_, which is what we want here.